### ISSUE TITLE
build all js files from source folder to target folder with its same name 

### ISSUE THREAD

**Issue by garyliu114** [markstory/asset_compress#336]
I have a lot of js files in webroot/js folder.
And I want to build js files separately for each.
is there any to do this in easily with a configuration file?
I want to like this.
[*.js]
files[] = *.js

**Comment by markstory** [comment]
No, there isn't right now. Without knowing the targets and the dependencies AssetCompress can't do tasks like automatic cache invalidation, build target validation and recursive builds.
